apps/bfd-model/bfd-model-codegen/src/main/java/gov/cms/bfd/model/codegen/MappingSpec.java

```diff
@@ -311,7 +332,9 @@ public String getLineEntityLineNumberField() {
    *     that should be used for the identifying yearMonth, if any
    */
   public String getEntityBeneficiaryMonthlyField() {
-    if (!isBeneficiaryEntity) throw new IllegalStateException();
+    if (!isBeneficiaryEntity) {
+      throw new IllegalStateException();
+    }
```

Comment:
these types of changes seem to be the only type in this file. any reason why this is changing on this PR?

Reply:
Apart from being grievously affronted by the lack of brackets? Personally. I believe this is an absolute line in the sand as far as code is concerned. Coming from background in other langs (espec c/c++) I have seen too many errors because devs miss things when not absolutely scoped. So my linter barks at this sort of stuff...

Reply:
I guess that last part is what I don't understand -- this class has no changes other than this cleanup that I can see.

I'm all for cleaning up stuff but I think the approach should be:

1. propose a style rule
2. get buy-in from the team and document the result in the style guide
3. modify the formatter and linter accordingly so that the change can be applied globally and not regress later
4. apply the change to the entire codebase at once in a single standalone PR

Reply:
what's the plan on this comment? Looks like all other comments resolved and this is about ready to go

Reply:
reverted back to no brackets.